src/coreclr/nativeaot/Runtime/eventtrace.cpp

```diff
@@ -189,7 +198,7 @@ void EtwCallbackCommon(
     // This callback gets called on both ETW/EventPipe session enable/disable.
     // We need toupdate the EventPipe provider context if we are in a callback
     // from EventPipe, but not from ETW.
-    if (isEventPipeCallback)
+    if (Change == EventPipeSessionEnable || Change == EventPipeSessionDisable)
```

Comment:
Can you update the 0/1 magic constants at line 208 just below as well?

Reply:
Done! Thanks for the catch.